opencti-platform/opencti-graphql/src/modules/attributes/stixDomainObject-registrationAttributes.ts

```diff
@@ -95,7 +95,7 @@ const stixDomainObjectsAttributes: { [k: string]: Array<AttributeDefinition> } =
     { name: 'objective', label: 'Objective', type: 'string', format: 'short', mandatoryType: 'no', editDefault: false, multiple: false, upsert: false, isFilterable: true },
   ],
   [ENTITY_TYPE_CONTAINER_NOTE]: [
-    { ...created, mandatoryType: 'external', editDefault: true, label: 'Publication date' },
```

Comment:
I'm unsure of this modification. It was a mistake from the start ? I just want to be sure it was not intended to have a specific label for `created` in Notes.

Reply:
Yes the label was different, but checked with @RomainGUIGNARD to uniformize the created attribute translation with with the other entity types

Reply:
What's more, the publication date filter is available for Notes, but Note overview it is displayed as 'original creation date'